apps/triggers/src/collaborator.status.ts

```
@@ -0,0 +1,68 @@
+          return;
+        }
+
+        const data = await response.json();
+
+        if (data.length === 0) {
+          break;
+        }
+
+        contributors = contributors.concat(data);
+        page += 1;
+      } while (true);
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)